src/py/setup.py

```diff
@@ -1,5 +1,18 @@
 # Needed for editable install
-import setuptools
+import os
 
-if __name__ == "__main__":
-    setuptools.setup()
+from setuptools import setup
+
+
+def find_stubs(package):
+    stubs = []
+    for root, _dirs, files in os.walk(package):
+        for file in files:
+            path = os.path.join(root, file).replace(package + os.sep, "", 1)
```

Comment:
Can you explain why we are doing the replace here?

Reply:
I copied this from the typeshed pep.

Reply:
OK, but what does it do ? ^^

I mean we have a single `js-stubs/__init__.pyi` file so do we actually need this?

Reply:
This could probably be simplified a lot yeah. I think just putting in a pathlib glob would be a good idea since we might want to add more typeshed files in the future and it would confuse people if they weren't included...

Reply:
Yes. Also  I'm a bit confused that this replace seems to be constructing a path of the form,
```
../py/__init__.pyi
```
which doesn't  exist. Or at least maybe it's a side effect of `os.walk`, if we go with pathlib glob, it might just ben,
```
package_data = {"js-stubs": Path(package).glob("**/*")}
```